iidm/iidm-mergingview/src/test/java/com/powsybl/iidm/mergingview/ThreeWindingsTransformerAdapterTest.java

```diff
@@ -132,7 +133,13 @@ public void testSetterGetter() {
         assertNotNull(leg3);
         assertTrue(leg3 instanceof AbstractAdapter);
 
-        // Not implemented yet !
-        TestUtil.notImplemented(twt::remove);
+        // Remove
+        leg1.getRatioTapChanger().remove();
+        assertNull(leg1.getRatioTapChanger());
+        leg2.getPhaseTapChanger().remove();
+        assertNull(leg2.getPhaseTapChanger());
```

Comment:
Do you also test the listener with these lines?

Reply:
After debuging tests, it appears it does not work.
I will fix that